### ISSUE TITLE
Layout Customer page

### ISSUE THREAD

**Issue by pembertonc** [pembertonc/gs.github.io#68]
Create the Customer page in accordance with the sheared document showing the appearance of pages

**Comment by pembertonc** [comment]
@akiferrol ,  Additionally you should enclose each label, input peer in a set of div s.  eg
<div> <label>street 1</label>
<input class="w3-input" type="text"  placeholder="Street address"/>
</div>

**Comment by pembertonc** [comment]
The editor will be used to edit and create new Customer.
It must be able to save and cancel changes.

**Comment by pembertonc** [comment]
@akiferrol ,  good moring,  Your back at school correct and hence unavailable is that correct.?

**Comment by akiferrol** [comment]
Yes I am back at school. Also, I will be starting work soon. Hence, I will be unavailable.
